tests/infra/consortium.py

```diff
@@ -347,6 +347,14 @@ def add_user(self, remote_node, user_id, user_data=None):
         proposal = self.get_any_active_member().propose(remote_node, proposal)
         return self.vote_using_majority(remote_node, proposal, careful_vote)
 
+    def create_and_withdraw_large_proposal(self, remote_node):
```

Comment:
I'd suggest adding a short comment here to explain why someone would ever want to use this.